bio/ngscheckmate/ncm/wrapper.py

```
@@ -0,0 +1,136 @@
+            )
+
+            # Only ncm_fastq.py handles multithreading
+            extra += f" --maxthread {snakemake.threads} "
+        else:
+            raise ValueError("Mixed BAM/VCF/Fastq. Please provide only one file type.")
+
+    list_file = snakemake.input.get("list_file", list_file)
+    extra += f" --list {list_file} "
+
+    cmd = (
+        f"export NCM_REF='{snakemake.input.fasta}' && "
```

Ruff (0.7.0)

Undefined name `snakemake`

(F821)